tests/subsys/net/lib/nrf_cloud/cloud/prj_qemu_cortex_m3.conf

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# ZTEST with new API
+CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
+
+# Network
+CONFIG_NETWORKING=y
+
+# Disable sockets and POSIX wrappers
+CONFIG_NET_SOCKETS=n
+CONFIG_NET_SOCKETS_POSIX_NAMES=n
+
+# Dependencies
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
+
+# qemu_cortex_m3
```

Comment:
Have a more descriptive comment than this. May be add a reference to https://github.com/zephyrproject-rtos/zephyr/issues/15565 ?

Reply:
changed to "# For qemu_cortex_m3 to run successfully, it is a fix in zephyrproject-rtos/zephyr#15565"